code/obj/item/gun/energy.dm

```diff
@@ -214,26 +216,194 @@ TYPEINFO(/obj/item/gun/energy)
 			projectiles += current_projectile
 
 
-////////////////////////////////////// Antique laser gun
-// Part of a mini-quest thing (see displaycase.dm). Typically, the gun's properties (cell, projectile)
-// won't be the default ones specified here, it's here to make it admin-spawnable (Convair880).
+//Antique laser gun
+
+//Maximum alpha a lens can be to fall into one of the damage tiers
+/// 180. glass, molitz, some other stuff
+#define CAPTAINGUN_T1_LENS_ALPHA_THRESHOLD 180
+/// 130. telecrystal, ectofiber, ice
+#define CAPTAINGUN_T2_LENS_ALPHA_THRESHOLD 130
+/// 80. starstone, rarely miraclium
+#define CAPTAINGUN_T3_LENS_ALPHA_THRESHOLD 80
+
+//Coil efficacy is scored as electrical conductivity + 2 if the material is an energy source
+//Minimum efficacy a coil can be to fall into one of the charge usage tiers
+/// 6. copper, pharosium, gold, etc.
+#define CAPTAINGUN_T1_COIL_EFFICACY_THRESHOLD 6
+/// 8. claretine, electrum, cerenkite, etc.
+#define CAPTAINGUN_T2_COIL_EFFICACY_THRESHOLD 8
+/// 10. custom alloy (figure it out, nerd)
+#define CAPTAINGUN_T3_COIL_EFFICACY_THRESHOLD 10
+
+//Damage dealt by different tiers of lens used
+/// 30. 2/3x normal laser damage
+#define CAPTAINGUN_T1_DAMAGE 30
+#define CAPTAINGUN_T1_DAMAGE_PROJECTILE_ICON_STATE "lasergat_laser"
```

Comment:
This is a bad way of doing this, make subtypes instead of just big lists of defines.